components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/service/api/GeoLocationBasedService.java

```diff
@@ -295,6 +295,60 @@ Response createGeoAlerts(
                     required = true)
             @PathParam("alertType") String alertType);
 
+
+    /**
+     * Create Geo alerts for geo clusters
+     */
+    @POST
+    @Path("/alerts/{alertType}")
+    @ApiOperation(
+            consumes = "application/json",
+            produces = "application/json",
+            httpMethod = "POST",
+            value = "Create Geo alerts for geo clusters",
+            notes = "",
```

Comment:
This is an optional element. If you don't have any notes correspond to the 'notes' field of the operation, you can remove it.

Reply:
Fixed by adding notes. c653051